src/app/sidebar/sidebar.component.html

```diff
@@ -1,7 +1,7 @@
 <div id="explorer-sidebar">
     <div class="arrow-left"></div>
     <div>
-        <span id="explorer-title" class="c-heading-3 panel-header" tabindex="0" [attr.title]="getStr('Graph Explorer')">
+        <span id="explorer-title" class="c-heading-3 panel-header" tabindex="-1" [attr.title]="getStr('Graph Explorer')">
```

Comment:
I don't see any changes giving programmatic focus to explorer-title so I think it makes sense to delete the tabIndex attribute here.